tests/bench/storage/utils.go

```diff
@@ -200,89 +198,31 @@ func runStorageBenchPut(
 	return nil
 }
 
-func runStorageBenchPutMany(
```

Comment:
question: What was wrong with this benchmark?

Reply:
corekv does not support Batching, it was benchmarking something that no longer exists (and wouldn't live in this repo anyway).